examples/google-esp-integration/server_only/README.md

```diff
@@ -0,0 +1,60 @@
+# Server-Only UID2 Integration Example
+
+This example demonstrates how a content publisher can use the UID2 services to implement the [custom UID2 integration workflow](https://github.com/UnifiedID2/uid2docs/blob/main/api/v1/guides/custom-publisher-integration.md).
+
+For an example application using the [Client-Side Identity JavaScript SDK](https://github.com/UnifiedID2/uid2docs/blob/main/api/v1/sdks/client-side-identity-v1.md), see [UID2 SDK Integration Example](../standard/README.md).
```

Comment:
Is there an ESP integration example which uses JS SDK?

Reply:
Yes, but only with SDK v3

Reply:
All docs are updated in another branch, which I will update them again once this PR get merged